torch/csrc/jit/codegen/cuda/dynamic_type.h

```diff
@@ -169,21 +180,10 @@ class TORCH_CUDA_CU_API IntOrDouble {
     return IntOrDouble(-as<double>());
   }
 
-  template <typename T>
-  bool operator==(T val) const {
-    return operator==(IntOrDouble(val));
-  }
-
-  template <typename T>
-  bool operator!=(T val) const {
-    return operator!=(IntOrDouble(val));
-  }
```

Comment:
Wasn't this just added? do we want to get rid of it?

Reply:
This is covered by previous line
```c++
  DEFINE_COMPARE_OP(==)
  DEFINE_COMPARE_OP(!=)
```